clients/message-client/src/openapi.json

```diff
@@ -30,8 +30,17 @@
         "operationId": "sendMessage",
         "summary": "sendMessage",
         "description": "Send an email message",
-        "tags": [
-          "Messages"
+        "tags": null,
+        "parameters": [
+          {
+            "in": "query",
+            "name": "do_not_create_entities",
+            "description": "When true, this flag lets the caller to send only the message and by-pass creating the entities.",
```

Comment:
Why entities (plural) ? What entities normally get created when sending a message. I would've assumed it's just a single message entity.

Reply:
It's also the thread entity we are creating while sending a message

Reply:
Got it! Then maybe I'd suggest mentioning these specific entities in the description.

I was immediately wondering if Message API auto-creates contact / file entities too

Reply:
sure thing 👍